mmv1/products/compute/NetworkFirewallPolicyWithRules.yaml

```diff
@@ -76,6 +76,16 @@ properties:
   - name: 'description'
     type: String
     description: An optional description of this resource.
+  - name: 'policyType'
```

Comment:
We can see if the tests catch it, but per the comment above, should any of these other fields be `default_from_api`? If they don't have `default_from_api` and the server provides a default value, I think a diff will result, which is especially bad because the fields are immutable.